ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-workflow.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-workflow.yaml",
+	"path": "cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-workflow.yaml",
```

Comment:
I think we should create a dedicated folder `fips` under `workers-rhel8` to be more clear.